includes/wikia/GlobalTitle.php

```diff
@@ -835,6 +844,32 @@ private function loadNamespaceNames() {
 	}
 
 	/**
+	 * Get $wgExtraNamespaces for title language
+	 *
+	 * @return array
+	 */
+	private function getLocalWgExtraNamespaces() {
+		global $wgExtensionNamespacesFiles;
+		$extraNamespaces = [ ];
+
+		foreach ( $wgExtensionNamespacesFiles as $extFile ) {
+			$namespaces = [ ];
+			// Namespace files fill in $namespaces array
+			require( $extFile );
+
+			foreach ( $namespaces as $langKey => $namespaceArray ) {
```

Comment:
Do we need to loop over all languages in the namespace array since we only want the values for `$this->mLang` and `'en'`?